setup.py

```diff
@@ -39,11 +39,11 @@ def _dbt_duckdb_version():
     install_requires=[
         "dbt-common>=1,<2",
         "dbt-adapters>=1,<2",
-        "duckdb>=0.7.0",
+        "duckdb>=0.7.0,!=0.10.3",
         # add dbt-core to ensure backwards compatibility of installation, this is not a functional dependency
         "dbt-core>=1.8.0",
     ],
-    extras_require={"glue": ["boto3", "mypy-boto3-glue"], "md": ["duckdb>=0.10.2"]},
+    extras_require={"glue": ["boto3", "mypy-boto3-glue"], "md": ["duckdb==0.10.2"]},
```

Comment:
I think this change isn't needed? The constraint on the top-level dependency should prevent 0.10.3 from being installed if the user requests `dbt-duckdb[md]`.

Reply:
Maybe not, but I don't think it does any harm

Reply:
This is only used by the CI right now